### PR TITLE
MINOR: Add random aborts to system test transactional copier service

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

**Comment by hachikuji** [comment]
cc @apurvam @ijuma

**Comment by hachikuji** [comment]
Passing here: http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2017-06-14--001.1497460918--hachikuji--add-random-aborts-to-system-test--b55ff7d/report.txt.

**Review by apurvam — APPROVED** [review]
LGTM! thanks!

File: tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java [hunk 1]
```diff
@@ -264,12 +272,14 @@ public void run() {
         });
 
         try {
+            Random random = new Random(System.currentTimeMillis());
```
Comment (ijuma) [inline tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java]: Hmm, what's the reasoning for using this random seed? The default uses `System.nanoTime` combined with some other information. Also, I wonder if we should be printing the seed used?

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM. Just a minor comment.

File: tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java (hunk 1 above)
Comment (hachikuji) [reply tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java]: Ack. We can use the default. I doubt we will make much use of the seed if it were printed. The logs would have to be pretty bad to need to resort to generating the sequence of commit/abort calls.

Comment (ijuma) [reply tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java]: Sounds good.

**Comment by hachikuji** [comment]
Merging to trunk and 0.11.0.

**Pull request closed without merging**